### PR TITLE
Fix suma API URL.

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/14903 - partial fix
orthography normalization (drive-by edit)

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by juliogonzalez — APPROVED** [review]
Same as the other PR. LGTM, but check if there are other links to the API at the doc, just in case :-)

File: .changelog [hunk 1]
```diff
@@ -6,3 +6,4 @@
 
 # Last packaged: 2021-05-14 10:00
 
+- Fixed URL of API documentation
```
Comment (juliogonzalez) [inline .changelog:9]: Add the bsc.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: .changelog (hunk 1 above)
Comment (juliogonzalez) [reply .changelog:9]: Ignore. I didn't create one. Approving again.

**Review by juliogonzalez — APPROVED** [review]
As long as you checked that this the only place where we link the API doc, LGTM.

**Comment by keichwa** [comment]
> Same as the other PR. LGTM, but check if there are other links to the API at the doc, just in case :-)

this seems to be the only place.

**Pull request merged**